build.sbt

```diff
@@ -1,8 +1,8 @@
 lazy val commonSettings = Seq(
   organization := "com.meetup",
-  version := "0.1",
+  version := "0.2",
   licenses += ("MIT", url("http://opensource.org/licenses/MIT")),
-  scalaVersion := "2.10.5",
+  scalaVersion := "2.11.7",
```

Comment:
By simply adding the setting `crossScalaVersions := Seq("2.10.5", "2.11.7")`, the project can be cross compiled to both versions. We'll need to test this against the older version of dispatch, but if it doesn't work we can probably work around it with some sbt fu.